main/py-django/APKBUILD

```diff
@@ -3,16 +3,16 @@
 pkgname=py-django
 _pkgname=Django
 pkgver=1.11
-pkgrel=0
+pkgrel=1
 pkgdesc="A high-level Python Web framework"
 url="http://djangoproject.com/"
 arch="noarch"
 license="BSD"
-depends="python2"
+depends=""
```

Comment:
Thanks for suggestion. It seems that at least not so few packages providing py3 and py2 subpackages use empty but present `depends` variable. For example main/py-yaml, main/py-tz, py-asn1. For now I will leave it as is.